draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -659,35 +659,62 @@ the claims `given_name`, `family_name`, and `address`, as it would be sent from
 
 # Verification and Processing {#verification}
 
-## Processing by the Holder  {#holder_verification}
+## Verification of the SD-JWT {#sd_jwt_verification}
+
+Upon receiving an SD-JWT, the Holders and the Verifiers MUST ensure that
+
+ * the issuer-signed JWT is valid, i.e., it is signed by the Issuer and the signature is valid, and
+ * all Disclosures are correct, i.e., their digests are referenced in the issuer-signed JWT.
 
-The Holder MUST perform the following (or equivalent) steps when receiving
-a Combined Format for Issuance:
+The Holder and the Verifier perform the following (or equivalent) steps when receiving
+an SD-JWT:
+
+ 1. Separate the SD-JWT into the issuer-signed JWT and the Disclosures (if any).
```

Comment:
```suggestion
 1. Separate the SD-JWT into the issuer-signed JWT, the Disclosures (if any), and the Key Binding JWT (if present). If a Key Binding JWT is received by a Holder, it MUST be ignored.
```
Not sure if this is the right rule, but we should probably say *something* on this.

Reply:
If the Holder received KB JWT, something is pretty wrong so probably such SD-JWT should be rejected?

Reply:
Works for me.